plugin/plugins/galgame_plugin/static/style.css

```
@@ -1188,229 +1829,1169 @@ button[disabled] {
+  border-radius: 12px;
+  background: rgba(255, 255, 255, 0.58);
+}
+
+.plugin-status-hub .status-grid-panel,
+.plugin-status-hub .data-grid,
+.plugin-status-hub .stack-list,
+.plugin-status-hub .result-panel,
+.plugin-status-hub .reply-text-scroll {
+  min-width: 0;
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)